### ISSUE TITLE
Min number or min percent of alt counts

### ISSUE THREAD

**Issue by mvelinder** [tonydisera/gene.iobio#792]
Have we ever thought of adding a filter for the minimum number (or percent) of alt alleles observed?

I see this sort of stuff all the time where there's very few alt counts and well below a nice 50/50 het split. This would allow user to "exclude variants with less than 5 alt counts" or "exclude variants with less than 20% alt counts"

![image](https://user-images.githubusercontent.com/22081541/43962501-1219642e-9c75-11e8-986e-8998a252d7aa.png)

**Comment by mvelinder** [comment]
Cool cool. Now we have an issue for it. Also nice to see it is an artifact that occurs in other datasets as well. Not just a UCGD/Senteion thing, but also happens in SFARI data.

Definitely would be utility in this.

**Comment by AlistairNWard** [comment]
Yeah, this is common. This was the impetus for my most recent push for getting pileup available as well. I found these cases, and clearly there's something going on the data, but I don't have UNIX access to the BAM files, so my investigation petered out.
